pkg/payerreport/workers/generator.go

```diff
@@ -232,6 +235,9 @@ func (w *GeneratorWorker) isOlderThanReportInterval(
 	// Convert the report's end minute since epoch to a time.Time
 	reportEndTime := time.Unix(int64(report.EndMinuteSinceEpoch)*60, 0).UTC()
 
-	// Check if the report is older than the minimum report interval
-	return time.Now().UTC().Sub(reportEndTime) > w.minReportInterval
+	if report.OriginatorNodeID == w.registrant.NodeID() {
```

Comment:
This change would generate a report for each node once per hour. Is that the intent of the change?

This function gets used to check if the last successfully submitted report is more than X hours old. If so, it's time to start creating new reports.

If every node reports on itself honestly and they get attestations, that would result in 24 payer reports per originator per day.